src/ExceptionHandling.java

```diff
@@ -32,12 +32,44 @@ public void getLength(String s){
             System.out.println("Exception cause : " + np.getMessage());
         }
     }
+
+    public int getArrayLength(int[] a){
+        try{
+            return a.length;
+        } catch (NullPointerException np){
+            System.out.println("Array is null : " + np.getMessage());
+        }
+        return 0;
+    }
+
+    // Making aware that this API could throw an exception. Possibly to be handled with try catch
+    public void getArrayIndexThrowException(int[] a, int index) throws ArrayIndexOutOfBoundsException {
+        System.out.println(a[index]);
+    }
+
+    //Handling the exception through try catch block
+    public void getArrayIndexHandleException(int[] a, int index) {
+        try{
+            System.out.println(a[index]);
+        } catch (ArrayIndexOutOfBoundsException e){
+            System.out.println(e.getMessage());
+        }
+    }
 }
 
 public class ExceptionHandling {
     public static void main(String[] args) {
         PerformOperations obj = new PerformOperations();
         obj.performDivision(100, 0);
         obj.getLength(null);
+        int[] arr = null;
+        int res = obj.getArrayLength(null);
+        System.out.println(res);
+        int[] testArray = new int[3];
+        testArray[0] = 1;
+        testArray[1] = 2;
+        testArray[2] = 3;
```

Comment:
Not sure why we are assigning array values in this fashion when the elements are sequential.

Reply:
addressed. Initialized normally without explicitly defining every element.